vscode-dotnet-runtime-extension/package.json

```diff
@@ -102,6 +102,11 @@
 				"dotnetAcquisitionExtension.allowInvalidPaths": {
 					"type": "boolean",
 					"description": "If you'd like to continue using a .NET path that is not meant to be used for an extension and may cause instability (please read above about the existingDotnetPath setting) then set this to true and restart."
+				},
+                "dotnetAcquisitionExtension.cacheTimeToLiveMultiplier": {

```

Comment:
This doesnt appear to be visible to the user.

Reply:
Apparently if the json is tab indented but space intended on this line then it does not show up in the ui 🤷